src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -263,9 +264,12 @@ public void testMissingAuthParam() throws Exception {
   @Test
   public void testInfo() throws Exception {
     final Info info = sut.info();
+    assertThat(info.driverStatus(), any(List.class));
```

Comment:
Yeah. Actually I guess it's a bad test anyway since it's basically testing whether type-safety is a thing.